packages/atomic/src/utils/initialization-utils.tsx

```diff
@@ -171,8 +173,9 @@ export function InitializeBindings<SpecificBindings extends AnyBindings>({
         }
       );
 
-      const canceled = element.dispatchEvent(event);
-      if (canceled) {
+      element.dispatchEvent(event);
+
+      if (!closest(element, initializableElements.join(', '))) {
```

Comment:
Do we have any kind of test for the "unhappy" path ?

ie: If I put an `atomic-facet` randomly outside of an interface, it should throw an error.

Reply:
Yes it's covered